project/profiles/capacity4more/modules/c4m/user/c4m_user_og/templates/c4m_unified_workflow_register.tpl.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Template to render the title of the registration form under unified workflow.
+ */
+?>
+<div class="c4m-unified-workflow-title">
+  <h1 class="title">
+    To join the <strong><?php print $group_title; ?></strong> group you need to register with capacity4more by filling in your details below.
+  </h1>
+  <h3 class="login">
+    Already a member? <?php print $login_link; ?>
+  </h3>
+</div>
```

Comment:
<img width="656" alt="screen shot 2016-07-13 at 11 32 02" src="https://cloud.githubusercontent.com/assets/789034/16798765/ed4abc24-48ed-11e6-95c2-a514df1752bb.png">

Few issues here:

1. This tpl is printed within an h1-tag, so we're nesting div/h1/h3 inside an h1-tag.
1. I think an h3 is more then sufficient
1. There's a lot of margin on top of it.